src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp

```diff
@@ -68,7 +69,16 @@ static std::shared_ptr<dnnl::convolution_forward::primitive_desc> get_convolutio
         auto is = input_md.get_dims()[2 + i];
         auto ks = weights_md.get_dims()[weights_offset];
         auto kernel_range = 1 + (ks - 1) * (dilation[i] + 1);
-        pad_r[i] = (os - 1) * stride[i] - is + kernel_range - pad_l[i];
+        auto padding = (os - 1) * stride[i] - is + kernel_range - pad_l[i];
```

Comment:
Should there be `- pad_l[i]` here?
It seems to imply that this is a total padding, since we then subtract from this value to obtain the l/r paddings
Or did I misunderstand?

Reply:
@Lyamin-Roman You are right! I updated it. Thanks!